lib/components/term.js

```diff
@@ -110,6 +116,17 @@ export default class Term extends PureComponent {
     this.fitResize();
   }
 
+  onWindowPaste(e) {
+    if (!this.props.isTermActive) return;
+
+    const processed = processClipboard();
```

Comment:
I'd like a comment to why this code is here to intercept the event, so it's easy to know why this is needed without following the code all the way

Reply:
Yep will do

Reply:
Perfect 👍